TouchID.m

```diff
@@ -6,16 +6,19 @@ @implementation TouchID
 
 RCT_EXPORT_MODULE();
 
-RCT_EXPORT_METHOD(authenticate: (RCTResponseSenderBlock)callback)
+RCT_EXPORT_METHOD(authenticate: (NSString *)reason
+                  callback: (RCTResponseSenderBlock)callback)
 {
     LAContext *context = [[LAContext alloc] init];
     NSError *error;
-    
+  if ( [[reason stringByTrimmingCharactersInSet:[NSCharacterSet whitespaceCharacterSet]] length] == 0 ){
```

Comment:
This is for people who wants to directly use the native module after implementing a change, like me. 

I think removing from the javascript will be a better option.